samples/angular/src/app/i18n/jss-translation-loader.service.ts

```diff
@@ -1,29 +1,18 @@
 import { Injectable } from '@angular/core';
 import { TranslateLoader } from '@ngx-translate/core';
-import { Observable } from 'rxjs';
-import { HttpClient } from '@angular/common/http';
+import { Observable, from as fromPromise } from 'rxjs';
+import { RestDictionaryService } from '@sitecore-jss/sitecore-jss-angular';
 import { environment as env } from '../../environments/environment';
-import { map } from 'rxjs/operators';
 
-class DictionaryResult {
-  phrases: any;
-}
+export const dictionaryService = new RestDictionaryService({
+  apiHost: env.sitecoreApiHost,
+  apiKey: env.sitecoreApiKey,
+  siteName: env.jssAppName,
+});
 
 @Injectable()
 export class JssTranslationLoaderService implements TranslateLoader {
-  constructor(
-    private http: HttpClient
-  ) { }
-
   getTranslation(lang: string): Observable<any> {
-    const options = {
-      params: {
-        sc_apikey: env.sitecoreApiKey
-      }
-    };
-    return this.http.get(`${env.sitecoreApiHost}/sitecore/api/jss/dictionary/${env.jssAppName}/${lang}`, options)
-      .pipe(
-        map((dictionary: DictionaryResult) => dictionary.phrases),
-      );
+    return fromPromise(dictionaryService.fetchDictionaryData(lang));
```

Comment:
I will create separate item, but I can't apply changes in fast way in each framework, because:
* React sample: it has internal 'i18next-fetch-backend' will need some time to remove it and use our service